feature-libs/product-configurator/rulebased/occ/variant/converters/occ-configurator-variant-normalizer.ts

```diff
@@ -257,6 +262,19 @@ export class OccConfiguratorVariantNormalizer
     images.push(image);
   }
 
+  getPlaceholder(
+    domainValues: OccConfigurator.Value[],
+    intervalInDomain: boolean
+  ): string {
+    let placeholder: string = '';
```

Comment:
I would rather like to have it initialized with undefined. In case place holder is not relevant, we should not put an empty string into the model

Reply:
fixed